packages/collector-proxy/package.json

```diff
@@ -13,26 +13,20 @@
   },
   "homepage": "https://github.com/rocket-connect/graphql-debugger#readme",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -b tsconfig.build.json",
     "test": "cross-env NODE_ENV=test jest --runInBand"
   },
   "peerDependencies": {
     "graphql": "^16.0.0 || ^17.0.0"
   },
   "dependencies": {
-    "@graphql-debugger/time": "workspace:^",
-    "@graphql-debugger/utils": "workspace:^",
     "@graphql-debugger/data-access": "workspace:^",
     "@graphql-debugger/trace-schema": "workspace:^",
-    "@graphql-debugger/queue": "workspace:^",
```

Comment:
What happens when we push to npm, will it say these packages are not found?

Reply:
Do these packages get shipped to `npm` too? I thought they'd be internal and just packaged into the final build